packages/react-components/src/organisms/CalendarList.tsx

```diff
@@ -100,19 +93,26 @@ const CalendarList: React.FC<CalendarListProps> = ({
                   </Paragraph>
                 </div>
                 <div css={gridButton}>
-                  <ExternalLink
-                    icon={plusIcon}
-                    label="Add to calendar"
-                    href={url.toString()}
-                  />
+                  <CalendarLink id={id} />
                 </div>
               </div>
             </li>
-          );
-        })}
-      </ul>
-    )}
-  </Card>
+          ))}
+        </ul>
+      )}
+    </Card>
+    <Paragraph accent="lead">
+      Having issues? Set up your calendar manually with these instructions for{' '}
+      <Link href="https://support.apple.com/en-us/guide/calendar/icl1022/mac">
+        iCal
```

Comment:
Umm, did apple seriously call their calendar `iCal`? I don't think so actually, isn't it `Apple Calendar` or so

Reply:
Yeah, they just call it Calendar.

Reply:
But...should we not ask design whether the copy should be changed then? :D

Reply:
Will they notice?